utils.py

```diff
@@ -176,52 +175,89 @@ def combine_vcf(combined_vcf, vcfs, duplicate_handling_mode = COMBINE_KEEP_ALL_D
                     output.write(previous_line)
         os.rename(uniq_vcf, combined_vcf)
     if gzip:
-        run_shell_command([BGZIP, "--force", combined_vcf], cmd_stdout=sys.stdout, cmd_stderr=sys.stderr)
-        index_vcf_gz(gz_vcf)
+        pysam.tabix_index(combined_vcf, force=True, preset='vcf')
         return gz_vcf
     else:
         return combined_vcf
 
-def index_vcf_gz(vcf_gz):
-    pysam.tabix_index(vcf_gz, force = True, preset = 'vcf')
 
-def sort_and_compress(vcf, mode = 1, overwrite = False):
+def sort_and_compress(vcf):
```

Comment:
Was the updated `utils.py` used here?

Reply:
I see utils.py was updated recently. I will fix this.